pkg/virtctl/templates/templates.go

```diff
@@ -0,0 +1,41 @@
+package templates
+
+// UsageTemplate returns the usage template for all subcommands
+func UsageTemplate() string {
+	return `Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+
+Flags:
+{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+
+Use "{{.CommandPath}} options" for a list of global command-line options (applies to all commands).{{end}}
```

Comment:
Either this doesn't read correctly, or the options command isn't configured correctly.

For example. You'll get something like this output to the cli.
```
Use "virtctl console options" for a list of global command-line options (applies to all commands)
```

But, the 'options' command is actually a top level command like 'console' and 'vnc'.

To get the options a user actually has to type

```
virtctl options console
``` 

instead of how it is documented as

```
virtctl console options
```

Reply:
Fixed